src/__tests__/displayAll.test.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable  */
```

Comment:
# Optional:
- To disable the linter is a bad practice. Please Linters helps to write clean code,